### ISSUE TITLE
S-58 Check 19 Re-classify from 'Warning' to 'Error'

### ISSUE THREAD

**Issue by iho-ohi** [iho-ohi/ENCWG-S58-ENC-Validation-Checks#1]
The AHO is of the idea that this validation check should be upgraded to an ERROR because it ‘may degrade the quality of the ENC through appearance’ (refer to s-58 section 1.2 Check Classification).
[ENCWG5_2020_11E_Use_of_the_edge_attribute_USAG_C_by_ECDIS_FINAL.pdf](https://github.com/iho-ohi/ENCWG-S58-ENC-Validation-Checks/files/5107088/ENCWG5_2020_11E_Use_of_the_edge_attribute_USAG_C_by_ECDIS_FINAL.pdf)

**Comment by FrankHippmann** [comment]
Agree

**Comment by IngaFjellanger** [comment]
We in Navico agree in changing Check 19 into giving an error, not warning.
We also agree that this should be looked into for S-64.

**Comment by TomRichardson6** [comment]
Agree

**Comment by alvarosanuy** [comment]
Agree.

**Comment by Christian-Shom** [comment]
Agree.

**Comment by JeffWootton** [comment]
Tend to agree, however I have a minor concern that reclassifying this as an error may prompt a Producing Authority to "blindly" amend their data to remove an error in validation where the outer boundary of the area is, in fact, coincident with the data limit of the cell.

**Comment by Christian-Shom** [comment]
Agree.
Following Jeff's comment, suggest to change the Check solution to read: 
"If the real world feature extends beyond the data limit of the cell, amend edge to USAG= 3 (exterior boundary truncated by the data limit)."

**Comment by TDYCARHugh** [comment]
I agree with Jeff that if this is an Error many will blindly set this when it is not necessary. There are for example meta features which are product specific (they do not extend beyond the product by definition) and thus would not be truncated.  This also seems like it is mainly a display issue and it might only be a concern for area features which display linestyles on their boundaries.

It would also be possible to make a check specifically for M_NSYS.

Note that the edge itself does not carry the USAG flag. This flag is carried on the feature object within the reference to a component edge.  An edge may truncate one feature but be a real boundary of another.
